django/db/models/query.py

```diff
@@ -563,9 +563,12 @@ def bulk_update(self, objs, fields, batch_size=None):
                     case_statement = Cast(case_statement, output_field=field)
                 update_kwargs[field.attname] = case_statement
             updates.append(([obj.pk for obj in batch_objs], update_kwargs))
+        rows_updated = 0
         with transaction.atomic(using=self.db, savepoint=False):
             for pks, update_kwargs in updates:
-                self.filter(pk__in=pks).update(**update_kwargs)
+                rows_updated += self.filter(pk__in=pks).update(**update_kwargs)
+
```

Comment:
```suggestion
```

Reply:
sorry, forgot to remove this one, have removed it now.